docs/Dashboard.md

```diff
@@ -0,0 +1,44 @@
+
+# Pulsar Dashboard
+
+Pulsar dashboard is a web application that let users monitor
+the current stats for all the topics, representing all
+the information in tabular form.
+
+The dashboard is composed of a "collector" that polls the
+stats from all the brokers in a Pulsar instance (across
+multiple clusters) and store all the information into
+a Postgres database.
+
+Finally a Django web-app is used to render the collected
+data.
+
+## Install
+
+The easiest way to use the dashboard is to run it inside
+a Docker container. A `Dockerfile` to generate the image
+is provided.
+
+To generate the Docker image:
+
+```shell
+$ docker build -t pulsar-dashboard dashboard
+```
+
+To run the dashboard:
+
+```shell
+$ MY_SERVICE_URL=http://broker.example.com:8080/
+$ docker run --rm -it -p 80:80 \
+    -e SERVICE_URL=$MY_SERVICE_URL\
+     pulsar-dashboard
+```
+
```

Comment:
You mean, where to point the browser to access the dashboard? Or from where the collector pulls that stats from?